apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeAggregateFeatureImportance.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getMenu } from "../../../../util/getMenu";
+import { modelAssessmentDatasets } from "../../modelAssessmentDatasets";
+
+import {
+  describeGlobalExplanationBarChart,
+  describeGlobalExplanationBarChartExplicitValues
+} from "./describeGlobalExplanationBarChart";
+import { describeGlobalExplanationBoxChart } from "./describeGlobalExplanationBoxChart";
+
+const testName = "Aggregate feature importance";
+
+export function describeAggregateFeatureImportance(
+  name: keyof typeof modelAssessmentDatasets
+): void {
+  const datasetShape = modelAssessmentDatasets[name];
+  if (datasetShape.featureImportanceData?.noFeatureImportance) {
+    return;
+  }
+  describe(testName, () => {
+    before(() => {
+      const hosts = Cypress.env().hosts;
+      cy.task("log", hosts);
+      cy.visit(hosts[0].host);
```

Comment:
I think this one would not work.
There are 3 notebooks started at same time, we probably need to based on file name and find coorposding host to visit here.

Reply:
Thanks for the comment, I agree, will update the code.